### PR TITLE
Fixes new linter errors

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
This fixes the errors raised by the new feature I'm adding to the linter: https://github.com/SpaceManiac/SpacemanDMM/pull/119

```
Analyzing proc bodies...

code/modules/reagents/chemistry/reagents/other_reagents.dm, line 218, column 2:
error: attempted PostIncr (++) on something with a type
code/modules/admin/verbs/SDQL2/SDQL_2.dm, line 685, column 5:
error: attempted PostIncr (++) on something with a type
code/modules/admin/verbs/SDQL2/SDQL_2.dm, line 692, column 5:
error: attempted PostIncr (++) on something with a type
code/modules/admin/verbs/SDQL2/SDQL_2.dm, line 714, column 6:
error: attempted PostIncr (++) on something with a type
code/datums/hud.dm, line 56, column 2:
error: attempted PreDecr (--) on something with a type
code/datums/hud.dm, line 92, column 3:
error: attempted PostIncr (++) on something with a type

Procs analyzed: 28647. Errored: 0. Builtins: 323.
```

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -214,15 +214,15 @@
 
 /datum/reagent/water/holywater/on_mob_life(mob/living/carbon/M)
 	if(!data)
-		data = 1
-	data++
+		data = list("misc" = 1)
+	data["misc"]++
```
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: why did you change this?

Comment (spookydonut) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: data is a list

Comment (optimumtact) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: tru

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 2]
```diff
@@ -715,7 +715,7 @@ GLOBAL_DATUM_INIT(sdql2_vv_statobj, /obj/effect/statclick/SDQL2_VV_all, new(null
 					SDQL2_TICK_CHECK
 					SDQL2_HALT_CHECK
 	if(islist(obj_count_finished))
-		obj_count_finished = obj_count_finished.len
+		obj_count_finished = length(obj_count_finished)
```
Comment (AnturK) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: Handling list case is kinda pointless here, it's not iterated or anything, might as well be simple number. I guess might be done in another PR.
